lib/titiler-pgstac-api/runtime/Dockerfile

```diff
@@ -1,20 +0,0 @@
-ARG PYTHON_VERSION
```

Comment:
@emileten we should try to keep this and use method described in https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_lambda_python_alpha/README.html#custom-bundling

Reply:
solution for this was to make things more configurable, in particular letting users pass `BundlingOptions`, however for the default we're not keeping the docker image. 

`BundlingOptions` allows to provide a custom bundling docker image, or docker commands to run after bundling (via `commandHooks`) -- which could be useful for trimming the dependencies like we were doing in this Dockerfile -- etc.. https://docs.aws.amazon.com/cdk/api/v2/docs/@aws-cdk_aws-lambda-python-alpha.BundlingOptions.html